lnrpc/rpc.proto

```diff
@@ -1498,6 +1498,11 @@ message ChannelEdge {
 }
 
 message ChannelGraphRequest {
+     /**
```

Comment:
All proto changes should be in its own commit :)

Reply:
Should I break it to the following commits?
I'm placing the proto changes first, as those are the ones that will most likely need to change upon rebase if master moves forward before the merge. is this the right consideration?

Mesage: "Add private property to ChannelGraphRequest rpc"
Files:  lnrpc/*

Message: "Add unit test for db Wipe"
Files:   channeldb/db_test.go

Message: "Add FetchPrivateOpenChannelIDs to easily check if an edge is private or public."
Files:  channeldb/db.go, channeld/channel_test.go

Message: "Add support for private property of ChannelGraphRequest in DescribeGraph rpc server method"  
Files: rpcserver.go, lnd_test.go

Message: " Add --private flag to describegraph lncli command.
    
    Private channels are returned only if the private flag is set.
    Note: this changes the default output of describegraph which
    used to return all channels known to node."
Files: cmd/lncli/commands.go

Thanks!

Reply:
Yes! Sounds like a perfect split! 👍